lesson-4/README.md

```diff
@@ -241,3 +241,27 @@ Next step is to add some dashboards which can be created manually but we will ju
 1. Discover provided graphs
 
 You can try to import [3662](https://grafana.com/dashboards/3662) and check Prometheus daemon usage.
+
+## Prometheus Node Exporter
+We can also use Prometheus to collect metrics of the nodes themselves.
+We use the **node exporter** for this. We can also use Kubernetes to deploy this
+to every node. We will use a [Kubernetes DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) to do so.
+
+In `node-exporter-daemonset.yaml` you will see that it looks similar to the
+deployment we did earlier. Notice that we run this in privileged mode `privileged: true`
```

Comment:
This will not work on properly secured cluster, but there is no alternative right? Maybe using host pid namespace etc can be enough?

Reply:
@dbecvarik fixed